src/test/integration_tests/api/iam/test_iam_basic_integration.js

```diff
@@ -32,33 +33,39 @@ const setup_options = { should_run_iam: true, https_port_iam: 7005, debug: 5 };
 coretest.setup(setup_options);
 const { rpc_client, EMAIL, get_current_setup_options, stop_nsfs_process, start_nsfs_process } = coretest;
 
-const CORETEST_ENDPOINT_IAM = coretest.get_iam_https_address();
-
 const config_root = path.join(TMP_PATH, 'test_nc_iam');
 // on NC - new_buckets_path is full absolute path
 // on Containerized - new_buckets_path is the directory
 const new_bucket_path_param = get_new_buckets_path_by_test_env(config_root, '/');
 
 let iam_account;
+let res;
```

Comment:
If you moved this variable as a global one, I think it would be better to give it a more meaningful name.